sdk/keyvault/azure-security-keyvault-secrets/vcpkg/Config.cmake.in

```diff
@@ -4,6 +4,7 @@
 @PACKAGE_INIT@
 
 include(CMakeFindDependencyMacro)
+find_dependency(azure-core-cpp "1.1.0")
```

Comment:
```suggestion
find_dependency(azure-core-cpp "1.2.0")
```

secrets is still not releasing, but it would be good to ask the last azure-core GA